usng2-js/build.gradle.kts

```diff
@@ -43,16 +41,25 @@ dependencies {
 configure<KotlinFrontendExtension> {
     downloadNodeJsVersion = "latest"
     sourceMaps = true
+    define("PRODUCTION", true)
```

Comment:
✏️ Probably doesn't matter at this size, but do we wan't to build production every time?

Reply:
For this project, I don't think it matters too much. For larger ones, it definitely will.